### ISSUE TITLE
Invalid “hours” in range cause cron #next_time loop breaker issue

### ISSUE THREAD

**Issue by ticky** [floraison/fugit#30]
## Issue description

Values outside 0-23 in an hour range cause an infinite loop error.

## How to reproduce

```ruby
require 'fugit'
c = Fugit.parse('* 0-24 * * *')
```

```
RuntimeError (too many loops for {:min=>0, :max=>23, :sta=>0, :edn=>24, :sla=>1} #range, breaking, please fill an issue at https://git.io/fjJC9)
```

## Expected behaviour

The “24” being out of range should either result in a parsing error or being treated as 0.

## Context

Please replace the content of this section with the output of the following commands:
```
Darwin Kinzie 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64
ruby 2.6.2p47 (2019-03-13 revision 67232) [x86_64-darwin18]
(secs:1565722428.8614998,utc~:"2019-08-13 18:53:48.8614997863769531",ltz~:"PDT")
(etz:nil,tnz:"PDT",tziv:"1.2.5",tzidv:nil,rv:"2.6.2",rp:"x86_64-darwin18",win:false,rorv:nil,astz:nil,eov:"1.2.1",eotnz:#<TZInfo::TimezoneProxy: America/Dawson>,eotnfz:"-0700",eotlzn:"America/Dawson",eotnfZ:"PDT",debian:nil,centos:nil,osx:"zoneinfo/America/Vancouver")
```

## Additional context

Encountered via in-the-wild user input!

**Comment by jmettraux** [comment]
Note in passing:

https://crontab.guru/#*_0-24_*_*_* says `24` is not in the allowed range.

**Comment by jmettraux** [comment]
Already accepting `24` on its own, https://github.com/floraison/fugit/blob/12a2faabc1d5f66fc8b4c8c3795d6b32c154ac2d/spec/cron_spec.rb#L50 so have to be lenient.

**Comment by jmettraux** [comment]
Released as part of fugit 1.3.2.

https://rubygems.org/gems/fugit/versions/1.3.2

Please tell me if there is anything wrong.

Many thanks with this issue report.

**Comment by ticky** [comment]
Thanks for the quick fix! Looking good 😄
